handlers/bounty.go

```diff
@@ -148,9 +149,24 @@ func DeleteBounty(w http.ResponseWriter, r *http.Request) {
 	}
 
 	b, _ := db.DB.DeleteBounty(pubkey, created)
+
+	w.WriteHeader(http.StatusOK)
 	json.NewEncoder(w).Encode(b)
 }
 
+func UpdatePaymentStatus(w http.ResponseWriter, r *http.Request) {
+	createdParam := chi.URLParam(r, "created")
```

Comment:
shouldnt we also check the pubkey of the bounty and the person sending the request?

Reply:
We removed the auth functionality for now, Once the auth is working perfectly I will add this to it, I will add a todo for now.

Reply:
gotcha sounds good!